xunit.runner.visualstudio.desktop/VsTestRunner.cs

```diff
@@ -286,54 +345,6 @@ static Stream GetConfigurationStreamForAssembly(string assemblyName)
         static TestProperty GetTestProperty()
             => TestProperty.Register("XunitTestCase", "xUnit.net Test Case", typeof(string), typeof(VsTestRunner));
 
-        List<AssemblyRunInfo> GetTests(IEnumerable<string> sources, LoggerHelper logger, IRunContext runContext, bool designMode)
-        {
-            // For store apps, the files are copied to the AppX dir, we need to load it from there
-#if WINDOWS_UAP
-            sources = sources.Select(s => Path.Combine(Windows.ApplicationModel.Package.Current.InstalledLocation.Path, Path.GetFileName(s)));
-#endif
-
-            var assemblyDiscoveredInfos = new List<AssemblyDiscoveredInfo>();
-
-            DiscoverTests(sources,
-                logger,
-                (source, discoverer, discoveryOptions) => new VsExecutionDiscoverySink(() => cancelled),
-                (source, discoverer, discoveryOptions, visitor) =>
-                {
-                    if (discoveryOptions.GetDiagnosticMessagesOrDefault())
```

Comment:
This logic and the filtering logic is moved to `RunTestsInAssembly` to allow reuse of same XunitFrontController for discovery and execution.